.github/workflows/ci-cd.yaml

```diff
@@ -75,19 +73,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-
-      - name: Fetch history
-        run: git fetch --prune --unshallow
-
+        with:
+          fetch-depth: 0
       - name: Configure Git
         run: |
           git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"          
```

Comment:
My OCD is getting triggered by that trailing empty space :smile:

Reply:
I totally blame VScode for this.